Cli 1.6 docs follow-up (#1289)
astronomer/docs: astro/cli/astro-dev-object-export.md

---
sidebar_label: "astro dev object export"
title: "astro dev object export"
id: astro-dev-object-export
description: Reference documentation for astro dev object export
---

> Export Ariflow variables, connections, and pools from a locally running environment to a file and format of your choice. 

Reviewer: ```suggestion
Export Airflow variables, connections, and pools from a locally running environment to a file and format of your choice. 
```

> Export Airflow variables, connections, and pools from a locally running environment to a file and format of your choice. 

Reviewer 2: ```suggestion
Export Airflow variables, connections, and pools from a locally running environment to a local file and format of your choice. You can choose to export all or some values by specifying flags to the command.

By default, all values will be exported to the `airflow_settings.yaml` file in your Astro project.
```

Author: IMO we don't need to mention the capabilities of flags in the overview descriptions of these commands

## Usage 

After starting your local Airflow environment with `astro dev start`, run:

```sh

> astro dev object export [flags]

Reviewer 2: Do we usually specify `[flags]` in these docs?

```

> By default, the command exports all variables, connections, and pools as YAML configurations to `airflow_settings.yaml`.

Reviewer 2: ```suggestion
```

## Options

| Option              | Description                                                                                                        | Possible Values             |
| ------------------- | ------------------------------------------------------------------------------------------------------------------ | --------------------------- |
| `-c`,`--connections` | Export connections to a given local file | ``|

> | `-e`,`--env`            | Location of the file to export Airflow objects to as Astro environment variables. Use this flag only with `--env-export`. The default file path is `env`.                                                                          | Any valid filepath                 |

Reviewer: ```suggestion
| `-e`,`--env`            | Location of the file to export Airflow objects to as Astro environment variables. Use this option only with `--env-export`. The default file path is `env`.                                                                          | Any valid filepath                 |
```

| `-n`,`--env-export`            | Export Airflow objects as Astro environment variables.                                                                                  | ``                 |
| `-p`,`--pools`            | Export pools to a given local file | ``                 |
| `-s`,`--settings-file`            | Location of the file to export Airflow objects to as YAML configuration. The default file path is `airflow_settings.yaml`.                                 | Any valid filepath              |

> | `-v`,`--variables`            | Export variables to a given local file | ``                 |

Reviewer 2: Again -- switch up order here?

- connections
- variables
- pools
- settings-file
- --env
- --env-export

Author: See above command - would prefer to keep alphabetical/ parallel with the CLI

> 

Reviewer 2: ```suggestion
```

## Examples 

```sh
astro dev object export --pools 

> # Exports pools from the local Airflow environment to `airflow_settings.yaml`

Reviewer 2: ```suggestion
# Exports only pools from the local Airflow environment to `airflow_settings.yaml`
```

astro dev object export --env-export --env="myairflowenv.env"
# Exports all Airflow objects from the local Airflow environment as 

> # Astro variables to a file in the project named `myairflowenv.env`

Reviewer 2: ```suggestion
# environment variables to a file named `myairflowenv.env`
```

Author: We should be more explicit that these variables are mainly meant for use on Astro

> ```

Reviewer 2: Related commands
